requirements_dev.txt

```diff
@@ -2,3 +2,4 @@ black==22.3.0
 isort==5.10.1
 pre-commit==2.19.0
 pylint==2.14.3
+PyGithub==1.55
```

Comment:
No need to add the package in two requirements files, just put it in the main one.